backend/api/tests.py

```
@@ -0,0 +1,43 @@
+from django.contrib.auth.hashers import make_password
+from rest_framework.test import APITestCase
+from rest_framework import status
+from .models import Users
+
+class AuthTestCase(APITestCase):
+
+    def setUp(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `setUp`

Add return type annotation: `None`

(ANN201)